src/ethereum/osaka/vm/instructions/system.py

```diff
@@ -1271,3 +1379,58 @@ def return_contract(evm: Evm) -> None:
 
     # PROGRAM COUNTER
     evm.running = False
+
+
+def pay(evm: Evm) -> None:
+    """
+    Transfer ether to an account.
+
+    Parameters
+    ----------
+    evm :
+        The current EVM frame.
+    """
+    # STACK
+    try:
+        to = to_address_without_mask(pop(evm.stack))
+    except ValueError as e:
+        raise ExceptionalHalt from e
+    value = pop(evm.stack)
+
+    # GAS
+    if to in evm.accessed_addresses:
+        access_gas_cost = GAS_WARM_ACCESS
+    else:
+        evm.accessed_addresses.add(to)
+        access_gas_cost = GAS_COLD_ACCOUNT_ACCESS
+
+    create_gas_cost = (
+        Uint(0)
+        if is_account_alive(evm.message.block_env.state, to) or value == 0
+        else GAS_NEW_ACCOUNT
+    )
+
+    transfer_gas_cost = Uint(0) if value == U256(0) else GAS_CALL_VALUE
+
+    charge_gas(evm, access_gas_cost + create_gas_cost + transfer_gas_cost)
+
+    # OPERATION
+    if value > U256(0):
+        try:
+            move_ether(
+                evm.message.block_env.state,
+                evm.message.current_target,
+                to,
+                value,
+            )
+            push(evm.stack, U256(1))
```

Comment:
Items being pushed to stack is not part of current spec (April 15) https://github.com/ethereum/EIPs/blob/5fae72b226be42a043e75c682d13d3eb889da17b/EIPS/eip-5920.md

An item being pushed is part of this proposed PR for the EIP: https://github.com/ethereum/EIPs/pull/9590

Note: it pushes 0 to stack if it is successful, and nonzero (1) in case it fails (so the other way around as of here). Motivation is in the EIP but this allows to add extra failure error codes in the future.

Reply:
The PR got changed and it now pushes 1 to stack on success and 0 on failure! So, we can say that EELS was ahead of the spec :smile: :+1: The PR got merged.